recipe/meta.yaml

```
@@ -1,34 +1,41 @@
 {% set name = "coin-or-utils" %}
-{% set version = "2.11.9" %}
+{% set version = "2.11.11" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/coin-or/CoinUtils/archive/releases/{{ version }}.tar.gz
-  sha256: 15d572ace4cd3b7c8ce117081b65a2bd5b5a4ebaba54fadc99c7a244160f88b8
+  sha256: 27da344479f38c82112d738501643dcb229e4ee96a5f87d4f406456bdc1b2cb4
   patches:
     - 0001-Patch-for-downstream.patch
 
 build:
   number: 0
-  skip: true  # [win]
   run_exports:
     - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
 
 requirements:
   build:
-    - gnuconfig  # [unix]
+    - gnuconfig
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}
-    - pkg-config
-    - make  # [unix]
+    - {{ compiler('fortran') }}  # [not win]
+    - pkg-config  # [not win]
+    - make  # [not win]
```

`# [unix]` is more correct than `# [not win]`; please prefer the former, especially if it's in use already (as here for `make`).

It would also be nice to align the selectors vertically.
```suggestion
    - make  # [unix]
```

---

I somehow always forget that MacOS is also Unix. Can do.